views/login_home.erb

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chitter</title>
+    <link rel="stylesheet" href="/style.css">
+  </head>
+  <body>
+    <h1>Chitter</h1>
+    <a href="/logout">Logout</a>
+    <h3>Start peeping now</h3>
+
+    <form action="/peeps" method="post">
+    
```

Comment:
Minor one - it'd be good to fix the indentation to improve the readability of the HTML structure (you can use any extension to auto-lint / format the code with your editor, to make things easier or automated whenever you save a file)